ex03.exs

```diff
@@ -51,11 +51,44 @@ defmodule Ex03 do
   """
 
   def pmap(collection, process_count, function) do
-    # your code goes here
-    # I'm hoping to see a simple pipeline as the body of this function...
+    makeList(collection)
+    |> splitMap(process_count)
+    |> spawnChunks(function)
+    |> getChunks
+    |> getResults
   end
 
-  # and here...
+  def makeList(collection) do
+    Enum.map(collection, fn x -> x end)
+  end
+
+  def splitMap(collection, process_count) do
+    split_count = div(length(collection), process_count)
```

Comment:
This `div` returns the wrong value (try it with 4 processors and 10 items).

# -2